java.sh

```diff
@@ -0,0 +1,33 @@
+package: Java
+version: "10.0.1"
+build_requires:
+ - curl
+---
+#! /bin/bash -e
+ZIP_NAME="jdk-10.0.1_linux-x64_bin.tar.gz"
```

Comment:
Do not hardcode 10.0.1, you can use `$PKGVERSION`, see [here](http://alisw.github.io/alibuild/reference.html).